tree/ntuple/v7/inc/ROOT/RNTupleMerger.hxx

```diff
@@ -19,6 +19,18 @@
 #include <ROOT/RError.hxx>
 #include <ROOT/RNTupleDescriptor.hxx>
 #include <ROOT/RNTupleUtil.hxx>
+#include <ROOT/RPageStorage.hxx>
+
+#include <memory>
+#include <mutex>
+#include <stdio.h>
+#include <string>
+#include <vector>
+#include <unordered_map>
+
+using RPageSink = ROOT::Experimental::Detail::RPageSink;
+using RPageSource = ROOT::Experimental::Detail::RPageSource;
+using RPageStorage = ROOT::Experimental::Detail::RPageStorage;
```

Comment:
`using` statement in header files affect all users of that header file.  For this reason we avoid such usage.  (But they are fair game in a source file)>

Reply:
Thanks a lot @pcanal. I dropped these in the latest version.